File: java/v1/src/main/java/com/api/v1/borrower/helpers/BorrowerResponseDto.java
package com.api.v1.borrower.helpers;

import java.time.LocalDate;

public record BorrowerResponseDto(
        String fullName,
        LocalDate birthDate,
        String ssn,
        String email,
        String address,
        String phoneNumber,
        String gender,
        String createdAt,
        String updatedAt
) {
}

File: koltlin/v1/src/main/kotlin/com/api/v1/borrower/helpers/BorrowerResponseDto.kt
package com.api.v1.borrower.helpers

import java.time.LocalDate

data class BorrowerResponseDto(
    val fullName: String,
    val ssn: String,
    val birthDate: LocalDate,
    val email: String,
    val address: String,
    val gender: String,
    val phoneNumber: String,
    val createdAt: String,
    val updatedAt: String
)
